packages/discord-bot/src/utils/praiseEmbeds.ts

```diff
@@ -211,7 +211,7 @@ export const notActivatedDM = async (msgUrl: string): Promise<MessageEmbed> => {
 
 export const selfPraiseWarning = async (): Promise<string> => {
   const msg = (await settingValue('SELF_PRAISE_WARNING')) as string;
-  if (msg && typeof msg === 'string') {
+  if (msg) {
```

Comment:
actually I think we don't need any of these `if(msg)` statements. settingValue will never return undefined -- instead it throws an error if the setting isn't found.

Reply:
So, should we add a try-catch block here?

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Inefficient

Reply:
> So, should we add a try-catch block here?

hmm, it depends how we want to handle the "default" value for `selfPraiseWarning`. If we want to keep it statically defined in the codebase as it is now, then yes wrapping it in a try-catch makes sense (and in the catch we would return the default value). 

Though I think it might be simpler to just remove the `selfPraiseWarning` wrapper entirely. If the setting is not defined it throws an error, which bubbles up to the handler function, which catches it and displays some user-facing message like "failed to forward praise".

does that make sense? It could also go in a separate PR as a more general refactor of the pattern in all the places it's used? Maybe that makes more sense.